clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -209,9 +209,14 @@ private EhcacheEntityResponse invokeInternal(EhcacheEntityMessage message, boole
     try {
       EhcacheEntityResponse response = waitFor(invokeAsync(message, replicate));
       if (Type.FAILURE.equals(response.getType())) {
+        /*
+         * The FAILURE cause is a server-side exception lacking client-side stack trace
+         * elements.  The server-side exception must be wrapped in a client-side exception
+         * to provide proper stack trace for analysis.
+         */
```

Comment:
This comment isn't need imho, that info is already captured in the javadoc of ClusteredEhcacheException.